src/pages/product-listing/ProductListing.jsx

```diff
@@ -23,37 +31,54 @@ const ProductListing = () => {
       priceRange ? Number(prod.price) <= Number(priceRange) : true
     );
 
-  const sortProds = (prods) => {
+  const ratingRangeFilter = (data) =>
+    data.filter((prod) =>
+      ratingRange ? Number(prod.rating) >= Number(ratingRange) : true
```

Comment:
I am not able to get why are using ternary over here you could have just written `Number(prod.rating) >= Number(ratingRange)` and that also would have worked fine I guess.

Reply:
It is because initially on first render i dont want the ratingRange filter to be applied at all, so initially i have given it a null value, so all the products will pass through the filter.